.travis.yml

```diff
@@ -46,6 +46,12 @@ jobs:
       script: npm run docs
       perl: '5.26'
 
+    # Commit Messages
+    -
+      before_install: git fetch --unshallow
+      env: LINT_COMMIT_MESSAGES=true
+      script: test/lint-commits.sh
```

Comment:
If you can run an npm script instead, which triggers the sh file, you could have commitlint in the `PATH`, simplifying the script a bit.

Reply:
Updated